src/main/java/com/aws/greengrass/clientdevices/auth/iot/Certificate.java

```diff
@@ -8,18 +8,77 @@
 import com.aws.greengrass.clientdevices.auth.session.attribute.AttributeProvider;
 import com.aws.greengrass.clientdevices.auth.session.attribute.DeviceAttribute;
 import com.aws.greengrass.clientdevices.auth.session.attribute.StringLiteralAttribute;
-import lombok.NonNull;
-import lombok.Value;
+import lombok.Getter;
 
+import java.time.Instant;
 import java.util.Collections;
 import java.util.Map;
 
-@Value
+@Getter
 public class Certificate implements AttributeProvider {
     public static final String NAMESPACE = "Certificate";
 
-    @NonNull
-    String iotCertificateId; // Needed for certificate revocation
+    public enum Status {
+        ACTIVE,
+        INACTIVE
+    }
+
+    String iotCertificateId;
+    Status status;
+    Instant lastUpdated;
+
+
+    // TODO: Needed?
```

Comment:
Will clean this up. Right now there are some tests which instantiate certificates directly